app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -524,7 +566,9 @@ class BrowserWebViewClient @Inject constructor(
                             title = navigationList.currentItem?.title,
                             start = safeStart,
                             end = currentTimeProvider.elapsedRealtime(),
-                            isTabInForeground = webViewClientListener?.isTabInForeground() ?: true,
+                            isTabInForegroundOnFinish = webViewClientListener?.isTabInForeground() ?: true,
+                            activeRequestsOnLoadStart = parallelRequestsOnStart,
+                            concurrentRequestsOnFinish = parallelRequestCounter.get() - 1,
```

Comment:
why not `decrementAndUntrackLoad` here, so that we have the actual number?

Reply:
I've updated the decrement function to do this in a single call.